oeps/processes/oep-0057-proc-core-product.rst

```diff
@@ -0,0 +1,356 @@
+
+* four Component Tiers: Kernel, Bundled Extensions, Official Extensions, and Unofficial Extensions.
+
+The community will use this common language to understand and classify the
+components of the Open edX product. These terms will provide a decision-making
+framework for product prioritization, feature support, and the contents of the
+community release. Below, we describe the meanings of each term.
+
+Product Narrative
+=================
+
+The *Product Narrative* is defined as the high-level vision statement for the
+Open edX project. It succinctly articulates the value proposition and impact of
+the project, the features which differentiate it, and the values and
+characteristics that position it as a market-leading solution. 
+
+Ongoing work to define the Product Narrative and subsequent deliverables can be
+tracked on the `Product Narrative wiki space`_.
+
+Product Offerings
+=================
+
+Core Product
+------------
+
+The *Core Product* is defined as a clear articulation of which features,
+experiences and use cases come fully supported by the default configuration of
+the community-maintained Open edX installation. The Core Product should be the
+community's first priority when it comes to maintenance, security patches, and
+feature investment.
+
+The Core Product is defined by a set of people across the community, driven by
+the `Product Working Group`_. It is based on user and market input. It answers key
+questions such as:
+
+* How do current and future features in the default Open edX installation align
+  with the Product Narrative and the needs of users?
+
+* What current and future features and experiences are available in the Core
+  Product, and why?
+
+* What is the spectrum of alternatives if a developed feature doesn't align with
+  the Core Product? 
+
+Ongoing work to define the Core Product and subsequent deliverables can be
+tracked on the `Core Product Offering wiki space`_.
+
+Extended Product(s)
+-------------------
+
+There has been interest in the community to define one or more *Extended
+Product(s)*. These hypothetical Product Offerings would enhance the Core Product
+with a set of extensions. An Extended Product might be created to, for example,
+support a specialized Open edX use case for which the Core Product alone is not
+suited.
+
+Although this OEP acknowledges the interest in such Product Offerings, there is
+currently no plan to fully define, implement, or offer any Extended Product. If
+such a Product Offering is codified in the future, this OEP will be updated.
+
+Full Catalog
+------------
+
+The *Full Catalog* is defined as the Core Product plus all known Open edX
+extensions.
+
+An actual catalog of components will be created and maintained to be as
+comprehensive as possible. Community members will be encouraged to submit their
+extensions to be listed in the Full Catalog.
+
+Component Tiers
+===============
+
+.. image:: oep-0057/oep-57-simplified-visual.png
```

Comment:
Whoops, in the wiki page we agreed to remove the "(Core Product and Open edX Extensions)" parenthetical underneath "Full Catalog" from this Lucid diagram, but that got resolved before we made the change.

I fixed this in the Lucid source; could you update the image here?